lib/services/local/discoveryconfig.go

```diff
@@ -118,3 +118,8 @@ func (s *DiscoveryConfigService) DeleteDiscoveryConfig(ctx context.Context, name
 func (s *DiscoveryConfigService) DeleteAllDiscoveryConfigs(ctx context.Context) error {
 	return trace.Wrap(s.svc.DeleteAllResources(ctx))
 }
+
+// UpdateDiscoveryConfigStatus updates the status of the specified DiscoveryConfig resource.
+func (s *DiscoveryConfigService) UpdateDiscoveryConfigStatus(ctx context.Context, name string, status discoveryconfig.Status) (*discoveryconfig.DiscoveryConfig, error) {
+	return nil, trace.NotImplemented("not implemented")
```

Comment:
Why is this not implemented? Please add a comment.

Reply:
Done in [687f1b0](https://github.com/gravitational/teleport/pull/40267/commits/687f1b01bcb6ae7469cd0661281eec135ac6f65a)